mpas_analysis/polar_regions.cfg

```diff
@@ -237,6 +250,41 @@ colorbarLevelsDifference = [-0.5, -0.2, -0.1, -0.05, -0.02, 0,  0.02, 0.05,
 regionGroups = ['Arctic Ocean Regions', 'Antarctic Regions']
 
 
+[profilesGreenlandRegions]
+## options related to plotting vertical profiles Greenland regions
+
+
+# a list of dictionaries for each field to plot.  The dictionary includes
+# prefix (used for file names, task names and sections) as well as the mpas
+# name of the field, units for colorbars and a the name as it should appear
+# in figure titles and captions.
+fields =
+    [{'prefix': 'potentialTemperature',
+      'mpas': 'timeMonthly_avg_activeTracers_temperature',
+      'units': r'$$\degree$$C',
+      'titleName': 'Potential Temperature'},
+     {'prefix': 'salinity',
+      'mpas': 'timeMonthly_avg_activeTracers_salinity',
+      'units': r'PSU',
+      'titleName': 'Salinity'},
+     {'prefix': 'potentialDensity',
+      'mpas': 'timeMonthly_avg_potentialDensity',
+      'units': r'kg m$$^{-3}$$',
+      'titleName': 'Potential Density'}]
+
+# Times for comparison times (Jan, Feb, Mar, Apr, May, Jun, Jul, Aug, Sep, Oct,
+# Nov, Dec, JFM, AMJ, JAS, OND, ANN)
+seasons =  ['ANN']
+
+# minimum and maximum depth of profile plots, or empty for the full depth range
+depthRange = []
```

Comment:
Should we impose a depth range of -1000 – 0 m here to be consistent with the TS diagrams? At least for the Matt's coupling work that's the depth range we mostly care about, but I'm not sure if there would be other E3SM folks interested in seeing deeper profiles off the Greenland shelf

Reply:
@alexolinhager Since we do not yet have the capability to actually mask the off-shelf quantities as of this PR, I think we should just keep this as-is and then change the range with the mask itself.

Reply:
Okay that makes sense to me!